microcosm_daemon/error_policy.py

```diff
@@ -2,9 +2,21 @@
 Error handling policy.
 
 """
+from logging import getLogger
+from time import time
+
 from microcosm.api import defaults
 
 
+# nagios style health codes
```

Comment:
I needed some basic health reporting to know that my test daemons were running properly. The logic here could easily become more sophisticated or be abstracted to another component. For now, it just logs health information at a regular interval and if anything changes.